ts/screens/wallet/FirstTransactionSummaryScreen.tsx

```diff
@@ -0,0 +1,261 @@
+/**
+ * This screen shows the transaction details.
+ * It should occur after the transaction identification by qr scanner or manual procedure.
+ * TODO: data displayed by this screen should be evaluated. The following data don't seem to be available:
+ *  - expire date
+ *  - cbill code
+ *  - telephone number
+ *  - web page
+ *  - pec
+ *  - email
+ *  - tranche
+ *  Data about the subject to which the transfer is releted to could be reviewed too.
+ */
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  H1,
+  H3,
+  Icon,
+  Left,
+  Right,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import AppHeader from "../../components/ui/AppHeader";
+import Modal from "../../components/ui/Modal";
+import PaymentSummaryComponent from "../../components/wallet/paymentSummary/PaymentSummaryComponent";
+import UpdatedPaymentSummaryComponent from "../../components/wallet/paymentSummary/UpdatedPaymentSummaryComponent";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+type State = Readonly<{
+  isModalVisible: boolean;
+  isAmountUpdated: boolean; // its value will depend on the match withe the amount on the notice and the remote value
+  amount: string;
```

Comment:
since this set of fields (`amount`, `expireDate`, `trance`, etc...) is used in multiple components we should probably define a type that describes it to avoid repeating the types